charts/primary-site/values.yaml

```diff
@@ -97,6 +97,15 @@ inboxListener:
       ## annotations:
       ##   eks.amazonaws.com/role-arn: arn:aws:iam::xxxxxxxxxxxx:role/foxglove-inbox-listener-sa-role
 
+  # To enable the autoscaling built into this chart, you must install KEDA first
+  # helm repo add kedacore https://kedacore.github.io/charts
+  # helm repo update
+  # helm install keda kedacore/keda --namespace keda --create-namespace
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 10
```

Comment:
Are these recommended values? Do we have any way to make recommendations?

Reply:
It depends on load and capacity, so we'll have to expound on this in docs. `False` and `1` are appropriate defaults in any case.